nilearn/_utils/docs.py

```diff
@@ -565,17 +565,28 @@ def custom_function(vertices):
 # bg_on_data
 docdict['bg_on_data'] = """
 bg_on_data : :obj:`bool`, optional
-    If ``True``, and a ``bg_map`` is specified,
+    If ``True`` and a ``bg_map`` is specified,
     the ``surf_data`` data is multiplied by the background
-    image, so that e.g. sulcal depth is visible beneath
-    the ``surf_data``.
+    image, so that e.g. sulcal depth is jointly visible with ``surf_data``.
+    Otherwise, the background image will only be visible where there
+    is no surface data (either because ``surf_data`` contains ``nan``s
```

Comment:
```suggestion
    is no surface data (either because ``surf_data`` contains ``nan``\s
```
This will resolve the doc build error. No need to do a full build again.
It's a tricky one. For explanation see [this](https://techwriter.documatt.com/restructuredtext-sphinx/pitfalls.html#rst-limit-not-separated-two-inlines)